backend/src/api/reports.ts

```diff
@@ -0,0 +1,54 @@
+import { wrapHandler } from './helpers';
+import S3Client from '../tasks/s3-client';
+import { getOrgMemberships } from './auth';
+
+/**
+ * @swagger
+ *
+ * /reports/export:
+ *  get:
+ *    description: Export CyHy report by specifying the S3 key returned in /reports/list.
+ *    tags:
+ *    - Reports
+ */
+export const export_report = wrapHandler(async (event) => {
+  const Key = JSON.parse(event.body!).Key;
+  const client = new S3Client();
```

Comment:
We need to do a check to ensure this key belongs to the orgs the user is a part of. Something like `getOrgMemberships(event).includes(orgId)`

Reply:
Maybe we just require two parameters in the body: orgId and report name. This way it's simpler and we don't need to worry about additional complexity with more validation.

Reply:
@aloftus23 checking to see if you could finish up this PR?

Reply:
Yes can get to it this afternoon

Reply:
Adressed in [cdc9769](https://github.com/cisagov/crossfeed/pull/1480/commits/cdc976942696c29b7618859076e1740b00622f58)